cpp/pipelines/add_authority_external_ref.cc

```diff
@@ -130,19 +181,164 @@ void ParseGndWikidataMappingFile(std::string filename,
         LOG_ERROR("input or output files could not be opened");
 }
 
+struct GNDStructure {
+    std::string gnd_id, wikidata_personal_entity_id, wikipedia_personal_address;
+};
+
+bool IsThisCloseBracketForId(const std::string &url) {
+    int url_length = url.length();
+    std::string is_about(url.substr(url_length - 5, 5));
+    if (is_about.compare("about") == 0)
+        return true;
+
+    return false;
+}
+
+bool DoesItMatch(const std::string &url_based, const std::string &url_comp) {
```

Comment:
That sounds very generic, can you please change the function name to be more specific?